config.json

```diff
@@ -216,6 +216,16 @@
         "pattern_matching",
         "stacks"
       ]
+    },
+    {
+      "slug": "isogram",
+      "uuid": "3c1b70fa-aaa8-4323-9aab-4c76da0a89e1",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
+        "strings"
```

Comment:
You could add logic and/or algorithms as topics

Reply:
Added both :)